src/test/java/org/cactoos/func/RetryTest.java

```diff
@@ -56,47 +55,18 @@ public void runsFuncMultipleTimes() throws Exception {
     }
 
     @Test
-    public void runsProcMultipleTimes() throws Exception {
-        MatcherAssert.assertThat(
-            new Retry<>(
-                input -> {
-                    if (new SecureRandom().nextDouble() > 0.3d) {
-                        throw new IllegalArgumentException("May happen");
-                    }
-                },
-                Integer.MAX_VALUE
-            ).apply(true),
-            Matchers.nullValue()
-        );
-    }
-
-    @Test
-    public void runsProcDefaultMultipleTimes() throws Exception {
-        final AtomicBoolean fail = new AtomicBoolean(true);
-        MatcherAssert.assertThat(
-            new Retry<>(
-                input -> {
-                    if (fail.getAndSet(false)) {
-                        throw new IllegalArgumentException("May happen");
-                    }
-                }
-            ).apply(true),
-            Matchers.nullValue()
-        );
-    }
-
-    @Test
-    public void runsProcConditionMultipleTimes() throws Exception {
+    public void runsFuncConditionMultipleTimes() throws Exception {
         MatcherAssert.assertThat(
             new Retry<>(
                 input -> {
                     if (new SecureRandom().nextDouble() > 0.3d) {
                         throw new IllegalArgumentException("May happen");
                     }
+                    return true;
                 },
                 count -> count == Integer.MAX_VALUE
             ).apply(true),
-            Matchers.nullValue()
+            Matchers.equalTo(true)
```

Comment:
@Iprogrammerr please use `FuncApplies`  instead of `EqualTo` and `Assertion` instead of `assertThat` from cactoos-matchers

Reply:
@victornoel We need to wait for #1023 to do that.